app/src/main/res/layout/activity_game_viewer.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:app="http://schemas.android.com/apk/res-auto"
+xmlns:tools="http://schemas.android.com/tools"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+tools:context=".activities.GameViewerActivity">
+
+<TextView
+    android:id="@+id/titleTextView"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="8dp"
+    android:layout_marginTop="8dp"
+    android:text="Title"
+    android:textSize="18sp"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent" />
+
+<TextView
+    android:id="@+id/gmTextView"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="8dp"
+    android:text="Game Master"
+    android:textSize="12sp"
+    app:layout_constraintStart_toStartOf="@+id/titleTextView"
+    app:layout_constraintTop_toBottomOf="@+id/titleTextView" />
+
+<TextView
+    android:id="@+id/descriptionTextView"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="8dp"
+    android:text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum semper ipsum et lacus congue, porttitor ullamcorper neque auctor. Aenean finibus ut lacus in vehicula. Nullam venenatis ex ultrices tellus maximus pharetra. Cras commodo vel lacus aliquam vehicula. Phasellus ullamcorper ut elit in imperdiet. Proin leo neque, consequat et commodo vitae, molestie eget ipsum. Duis cursus vestibulum consequat. Fusce cursus ligula ut augue gravida, sed hendrerit lacus sollicitudin. Cras pulvinar vulputate velit, ac efficitur odio ultricies at. Donec faucibus vitae quam nec scelerisque. Vestibulum volutpat ut dolor eu aliquam. "
```

Comment:
Is this necessary?

Reply:
(For it to be so long)

Reply:
It's a default text. That way I can see how a long description would look like while working on the design of the activity.